ods_ci/tests/Resources/OCP.resource

```diff
@@ -108,7 +108,7 @@ Get Events And Pod Logs
 
 Wait For Pods To Be Ready
     [Arguments]    ${label_selector}    ${namespace}    ${timeout}=600s    ${exp_replicas}=${NONE}
-    Wait Until Keyword Succeeds    ${timeout}    3s
+    Wait Until Keyword Succeeds    ${timeout}    15s
```

Comment:
why increasing the retry interval? was it giving some problem? I'm not against it, but curious about the reason

Reply:
Because when i was trying to run and test maually on local it was failing continuosly i thought it would help me , but the issue was from my laptops (infra issue) not the cluster or network